internal/executor/executor_test.go

```diff
@@ -371,6 +372,47 @@ func TestPersistentWorker(t *testing.T) {
 	assert.Contains(t, buf.String(), "'check' script succeeded")
 }
 
+func TestPersistentWorkerContainerIsolationVolumes(t *testing.T) {
+	// Prepare the directory that we're going to mount inside of the container
+	dirToBeMounted := testutil.TempDir(t)
+	if err := ioutil.WriteFile(filepath.Join(dirToBeMounted, "some-file-name.txt"), []byte{}, 0600); err != nil {
+		t.Fatal(err)
+	}
+
+	// Prepare the configuration that ensures that the mounted directory contains the file we've created above
+	config := fmt.Sprintf(`persistent_worker:
+  isolation:
+    container:
+      image: debian:latest
+      volumes:
+        - %s:/dir-to-be-mounted
+
+unix_task:
+  show_script: ls -lah /dir-to-be-mounted
+  check_script:
+    - test -e /dir-to-be-mounted/some-file-name.txt
```

Comment:
Maybe it's better to test that we can mount a folder and a task can write a file to it? Seems like a more common use case to just have a persistent folder that tasks are writing to. 🤔